tsdx.config.js

```diff
@@ -0,0 +1,16 @@
+// TSDX does not currently support multiple entry files out of the box,
```

Comment:
TSDX does not currently support multiple entry files out of the box, so I had to overwrite the Rollup config
Ref: https://github.com/formium/tsdx/issues/175#issuecomment-564814325